### PR TITLE
Calc cpu percent using last variables if interval of 0 is given

### LINKED ISSUE #202: cpu percent(0, true) fails to return expected results
the output is always either this:
```
[]float64) (len=16 cap=16) {
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0
}
```
or this
```
[]float64) (len=16 cap=16) {
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 100,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0,
 (float64) 0
}
```

previously if I would make multiple calls it would give me the percentage used between the two calls. Now it seems like it's giving me whatever it see's right when I call it.

What I am after here is a simple call I can make multiple times and each time it gives me the difference between the two calls.

### PR DESCRIPTION
**Pull request by vrecan**
This fixes #202

### CODE REVIEW

**Comment by vrecan** [comment]
I have this running in our internal app that collects our node stats and everything looks good. I also ran the tests with the race detector and all the proc tests pass. 

Side note: 
I do see a one test failing on my linux box:
```
--- FAIL: TestGetProcInodesAll (0.01s)
        Error Trace:    net_linux_test.go:20
	Error:  	Should NOT be empty, but was map[]		
```

Not related to what I changed but I thought you might want to know about it.

File: cpu/cpu.go [hunk 1]
```diff
@@ -2,11 +2,11 @@ package cpu
 
 import (
 	"encoding/json"
+	"github.com/shirou/gopsutil/internal/common"
```
Comment (KenjiTakahashi) [inline cpu/cpu.go]: Why mix these? Hint: Use `goimports` for the greater good :-).

File: cpu/cpu_unix.go [hunk 2]
```diff
@@ -7,24 +7,46 @@ import (
...
+	if t2Busy <= t1Busy {
+		return 0
+	}
+	if t2All <= t1All {
+		return 1
+	}
+	return (t2Busy - t1Busy) / (t2All - t1All) * 100
+}
+
+func calcualteALLBusy(t1, t2 []TimesStat) ([]float64, error) {
```
Comment (KenjiTakahashi) [inline cpu/cpu_unix.go]: Function name typo.

Comment (vrecan) [inline cpu/cpu_unix.go]: thanks, fixed

File: cpu/cpu.go (hunk 1 above)
Comment (vrecan) [inline cpu/cpu.go]: goimports never plays nice for me with remote mounting filesystems in sublime.

**Comment by shirou** [comment]
Great work! Thank you !!

I run tests several times but I could not get this error. So, I merged this PR. If someone meets this, please send information.
```
--- FAIL: TestGetProcInodesAll (0.01s)
        Error Trace:    net_linux_test.go:20
    Error:      Should NOT be empty, but was map[]  
``

**Pull request merged**